assets/src/hooks/use-copy-ticket.js

```diff
@@ -14,31 +14,34 @@ import useCreateRelationsForTicketToEventDates
 import useCreateRelationsForTicketToPrices
 	from './use-create-relations-for-ticket-to-prices';
 
+const PRICE = 'price';
```

Comment:
@tn3rb let me know if I can pull these somewhere in a shared location.
Or maybe it's not so good idea.